src/installLogsPrinter.js

```diff
@@ -68,17 +72,25 @@ function installLogsPrinter(on, options = {}) {
       }
 
       if (options.outputTarget) {
+        allMessages[0] = data.state;
```

Comment:
This is not correct. You are saving multiple test`s runs state on the same key: 0. This means you will only have access to the last state saved here.